services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/hakemus/HakemusService.kt

```diff
@@ -279,7 +276,8 @@ class HakemusService(
             hakemus.alluStatus = response.status
         }
 
-        logger.info("Sent hakemus. ${hakemus.logString()}, alluStatus = ${hakemus.alluStatus}")
+        logger.info(
+            "Sent hakemus ${hakemus.applicationType}. ${hakemus.logString()}, alluStatus = ${hakemus.alluStatus}")
```

Comment:
Would it be good to include applicationType in logString() function?

Reply:
Yes, that would be something we're often interested in when diagnosing an application problem. Added.